source/codegen/service_helpers.py

```diff
@@ -37,6 +37,49 @@ def create_args_for_ivi_dance(parameters):
         result = result + common_helpers.camel_to_snake(parameter['cppName']) + ', '
     return result[:-2]
 
+def create_args_for_ivi_dance_with_a_twist(parameters):
+    result = ''
+    for parameter in parameters:
+      name = common_helpers.camel_to_snake(parameter['cppName'])
+      is_array = common_helpers.is_array(parameter['type'])
+      if parameter.get('is_size_param', False):
+        result = f'{result}0, '
+      elif common_helpers.is_output_parameter(parameter):
+          if is_array:
+              result = f'{result}nullptr, '
+          else:
+              result = result + f'&{name}' + ','
+      else:
+        result = result + common_helpers.camel_to_snake(name) + ', '
+    return result[:-1]
```

Comment:
This should be `result[:-2]` to remove "," and " " from result. This was working so far because you were missing the space as per above comment.

Reply:
Done